### ISSUE TITLE
Development - Add dedicated command for creating element in new db

### ISSUE THREAD

**Issue by rompik** [eryar/PipeCAD#19]
I think will be more clear for developer if for creating SITE/DEPT inside new db will be done using command:
```python
aDbItem.CreateDbElement("X-XXX-001/CA")   #which will create only high level elements like SITE, DEPT
```
but not like:
```
aDbItem.CreateDatabase("X-XXX-001/CA")
```

**Comment by eryar** [comment]
Okay.

**Comment by eryar** [comment]
Fixed PipeCAD-1.0.9
